docs/usage-examples/findOne.txt

```diff
@@ -0,0 +1,118 @@
+collection in the ``sample_mflix`` database. The example uses the ``Movie`` Eloquent model
+to represent the ``movies`` collection.
+
+This example performs the following actions:
+
+- Defines a query filter that matches documents in which the value of the ``directors`` field
+  is ``"Rob Reiner"``
+- Retrieves the first document that matches the query filter, according to the sort order
+  specified by the ``orderBy()`` method
+
+To see the find operation code, select the :guilabel:`Controller File Code` tab. To see
+the HTML code that specifies a view, select the :guilabel:`View File Code` tab.
+
+.. tabs::
+
+   .. tab:: Controller File Code
+      :tabid: find-one-controller
+
+      .. io-code-block::
+         :copyable: true
+
+         .. input::
+            :language: php
+            
+            class MovieController
+            {
+                 public function show()
+                 {
+                      $movie = Movie::where('directors', 'Rob Reiner')
+                          ->orderBy('_id')
+                          ->first();
+
+                      return view('browse_movies', [
+                          'movie' => $movie
+                      ]);
+                  }
+            }
+
+         .. output::
+            :language: console
+            :visible: false
+
+            Title: This Is Spinal Tap
+            Year: 1984
+            Runtime: 82
+            Director: Rob Reiner
+            Plot: Spinal Tap, one of England's loudest bands, is chronicled by film director
+            Marty DeBergi on what proves to be a fateful tour.
+
+   .. tab:: View File Code
+      :tabid: find-one-view
+
+      .. io-code-block::
+         :copyable: true
+
+         .. input::
+            :language: php
+            
+            <!DOCTYPE html>
+            <html>
+            <head>
+            <title>Browse Movies</title>
+            </head>
+            <body>
+            <h2>Movies</h2>
+            <p>
+                Title: {{ $movie->title }}<br>
+                Year: {{ $movie->year }}<br>
+                Runtime: {{ $movie->runtime }}<br>
+                Director: {{ $movie->directors[0] }}<br>
+                Plot: {{ $movie->plot }}<br>
+            </p>
+            </body>
+            </html>
+
+         .. output::
+            :language: console
+            :visible: false
+
+            Title: This Is Spinal Tap
+            Year: 1984
+            Runtime: 82
+            Director: Rob Reiner
+            Plot: Spinal Tap, one of England's loudest bands, is chronicled by film director
+            Marty DeBergi on what proves to be a fateful tour.
+
+To perform the find operation, start your Laravel application by running the following command:
+
+.. code-block:: bash
+
+   php artisan serve
+
+Then, open the following URL in your web browser to view the expected output:
+
+.. code-block:: none
+
+   http://127.0.0.1:8000/browse_movies
```

Comment:
Q: Should we include a link to the full git blob as we do with other driver usage example such as [here](https://www.mongodb.com/docs/drivers/go/current/usage-examples/insertMany/#:~:text=%7D-,View%20a,-fully%20runnable%20example) in Go?
Just a question, we dont do this for some of the other drivers so I am indifferent

Reply:
I'm kind of against it in this case, since we'd have to link not just to one runnable file but to all the necessary Laravel files (Model, controller, view, etc)